cmd/snap/cmd_auto_import_test.go

```diff
@@ -93,3 +97,26 @@ func (s *SnapSuite) TestAutoImportAssertsHappy(c *C) {
 	c.Check(s.Stderr(), Matches, fmt.Sprintf("(?ms).*imported %s\n", fakeAssertsFn))
 	c.Check(n, Equals, total)
 }
+
+func (s *SnapSuite) TestAutoImportAssertsHappyNotOnClassic(c *C) {
+	restore := release.MockOnClassic(true)
+	defer restore()
+
+	fakeAssertData := []byte("my-assertion")
+	s.RedirectClientToTestServer(func(w http.ResponseWriter, r *http.Request) {
+		panic("not auto-import is run on classic")
```

Comment:
Yeah, it needs tweaking. I'll merge as that's just in the test and mvo is going to be out for a few days.